### ISSUE TITLE
[FEATURE REQUEST] Option to set custom API provider for external IP

### ISSUE THREAD

**Issue by ace2k** [ArcadeRenegade/SidebarDiagnostics#275]
### **Request:**
While there's nothing wrong with ipify.org, it would be nice to have the option to set a different/custom API provider. There are various API providers which offer decimal IPv4 plaintext output as required by SidebarDiagnostics.
Reference: [Constants.cs](https://github.com/ArcadeRenegade/SidebarDiagnostics/blob/master/SidebarDiagnostics/Constants.cs#L13)

### **Example APIs:**
Like the built-in [`https://api.ipify.org`](https://api.ipify.org) these public APIs work out of the box without an API key.
https://icanhazip.com
https://ifconfig.me/ip
https://ifconfig.co/ip
https://ipinfo.io/ip
https://bot.whatismyipaddress.com

### **Implementation:**  
The newbie-friendly method would be a drop-down menu with predefined entries such as above. The cool-kids method would be an input form with a hint on the required format. I'd prefer an input form but I'll leave this one to your judgement.

### **What's the point?**
Customizability, really.
What's the point of running SidebarDiagnostics? Custom visual swag for your desktop. So why stop here? Bonus; in the unlikely event of ipify being blocked on your local network for whatever reason, you would have the option to just hop to the next API provider without the trouble of a hardcoded, unchangeable variable.

Thanks for reading!
